packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1005,20 +1008,43 @@ export const VIDEO_RESOLUTIONS = {
         ideal: 720
       }
     }
+  },
+  '1080p': {
+    video: {
+      width: {
+        max: 1920,
+        ideal: 1920
+      },
+      height: {
+        max: 1080,
+        ideal: 1080
+      }
+    }
   }
 };
 
+export const VIDEO_RESOLUTIONS = {
+  [QUALITY_LEVELS.LOW]: AVALIABLE_RESOLUTIONS['480p'],
+  [QUALITY_LEVELS.MEDIUM]: AVALIABLE_RESOLUTIONS['720p'],
+  [QUALITY_LEVELS.HIGH]: AVALIABLE_RESOLUTIONS['1080p'],
+  [QUALITY_LEVELS['360p']]: AVALIABLE_RESOLUTIONS['360p'],
+  [QUALITY_LEVELS['480p']]: AVALIABLE_RESOLUTIONS['480p'],
+  [QUALITY_LEVELS['720p']]: AVALIABLE_RESOLUTIONS['720p'],
+  [QUALITY_LEVELS['1080p']]: AVALIABLE_RESOLUTIONS['1080p'],
+};
+
 /**
- * Max frame sizes based on h264 configs
+ * Max frame sizes and bandwith constraints based on h264 configs
```

Comment:
We don't have max-mbps in constraints, We can update this to 'Max frame sizes based on h264 configs', I will commit the changes.

Reply:
Updated the commented code.